packages/smooth_ui_library/lib/dialogs/smooth_AlertDialog.dart

```diff
@@ -13,72 +13,99 @@ import 'package:smooth_ui_library/buttons/smooth_simple_button.dart';
 ///
 
```

Comment:
Actually the file name should be normalized; more something like `smooth_alert_dialog.dart`